impl/e2e-tests/lookback.json

```diff
@@ -1,6 +1,8 @@
 {
-  "$comment": "use different advertiser sites to avoid depending on budgeting",
-  "$comment": "86400 seconds in a day",
+  "$comment": [
+    "use different advertiser sites to avoid depending on budgeting",
+    "86400 seconds in a day"
+  ],
```

Comment:
Is the array really necessary?  I know that shorter lines are good, but this seems like overkill